applications/UmatApplication/umat_models/TensorModule.f

```diff
@@ -4,8 +4,8 @@
 C     w-fuente@uniandes.edu.co
 C     Prof. Dr-Ing Arcesio Lizcano
 C     alizcano@uniandes.edu.co
-C     Grupo de Investigaci�n de Geotecnia de la Universidad de los Andes
-C     Universitiy Los Andes, Bogot� Colombia
+C     Grupo de Investigaci�n de Geotecnia de la Universidad de los Andes
+C     Universitiy Los Andes, Bogot� Colombia

```

Comment:
These characters don't render on my machine, so I suggest just getting rid of accents.

```suggestion
C     Grupo de Investigacion de Geotecnia de la Universidad de los Andes
C     Universitiy Los Andes, Bogota Colombia
```